src/Tribe/Aggregator/API/Origins.php

```diff
@@ -33,19 +33,19 @@ public function __construct() {
 			),
 			'eventbrite' => (object) array(
 				'id' => 'eventbrite',
```

Comment:
`esc_html_x()` for the ones that remain translatable (the ones that aren't brand names)